src/EnergyPlus/EcoRoofManager.cc

```diff
@@ -757,6 +540,204 @@ namespace EcoRoofManager {
+        SetupOutputVariable(state,
+                            "Green Roof Cumulative Irrigation Depth",
+                            OutputProcessor::Unit::m,
+                            thisEcoRoof->CumIrrigation,
+                            OutputProcessor::SOVTimeStepType::Zone,
+                            OutputProcessor::SOVStoreType::State,
+                            "Environment");
+        SetupOutputVariable(state,
+                            "Green Roof Cumulative Runoff Depth",
+                            OutputProcessor::Unit::m,
+                            thisEcoRoof->CumRunoff,
+                            OutputProcessor::SOVTimeStepType::Zone,
+                            OutputProcessor::SOVStoreType::State,
+                            "Environment");
+        SetupOutputVariable(state,
+                            "Green Roof Cumulative Evapotranspiration Depth",
+                            OutputProcessor::Unit::m,
+                            thisEcoRoof->CumET,
+                            OutputProcessor::SOVTimeStepType::Zone,
+                            OutputProcessor::SOVStoreType::State,
+                            "Environment");
+        SetupOutputVariable(state,
+                            "Green Roof Current Precipitation Depth",
+                            OutputProcessor::Unit::m,
+                            thisEcoRoof->CurrentPrecipitation,
+                            OutputProcessor::SOVTimeStepType::Zone,
+                            OutputProcessor::SOVStoreType::Summed,
+                            "Environment");
+        SetupOutputVariable(state,
+                            "Green Roof Current Irrigation Depth",
+                            OutputProcessor::Unit::m,
+                            thisEcoRoof->CurrentIrrigation,
+                            OutputProcessor::SOVTimeStepType::Zone,
+                            OutputProcessor::SOVStoreType::Summed,
+                            "Environment");
+        SetupOutputVariable(state,
+                            "Green Roof Current Runoff Depth",
+                            OutputProcessor::Unit::m,
+                            thisEcoRoof->CurrentRunoff,
+                            OutputProcessor::SOVTimeStepType::Zone,
+                            OutputProcessor::SOVStoreType::Summed,
+                            "Environment");
+        SetupOutputVariable(state,
+                            "Green Roof Current Evapotranspiration Depth",
+                            OutputProcessor::Unit::m,
+                            thisEcoRoof->CurrentET,
+                            OutputProcessor::SOVTimeStepType::Zone,
+                            OutputProcessor::SOVStoreType::Summed,
+                            "Environment");
+    }
+
+    void initEcoRoof(EnergyPlusData &state, int const SurfNum, int const ConstrNum)
+    {
+        // Using/Aliasing
+        using namespace DataEnvironment;
```

Comment:
With this small function it's not important, but I think it's always better to remove usings and just be explicit at the usage.